features/mailbox_specs/mailbox_spec.feature

```diff
@@ -0,0 +1,85 @@
+@rails_post_6
+Feature: action mailbox spec
+
+  Mailbox specs provide alternative assertions to those available in `ActiveMailbox::TestHelper` and help assert behaviour of how the email
+  are routed, delivered, bounced or failed.
+
+  Mailbox specs are marked by `type: :mailbox` or if you have set
+  `config.infer_spec_type_from_file_location!` by placing them in `spec/mailboxes`.
+
+  With mailbox specs you can:
+  * `process(mail_or_attributes)` - send mail directly to the mailbox under test for `process`ing.
+  * `receive_inbound_email(mail_or_attributes)` - matcher for asserting whether incoming email would route to the mailbox under test.
+  * `have_been_delivered` - matcher for asserting whether an incoming email object was delivered.
+  * `have_bounced` - matcher for asserting whether an incoming email object has bounced.
+  * `have_failed` - matcher for asserting whether an incoming email object has failed.
+
+  # Background:
```

Comment:
Ok, that's why the PR is in draft. I am no able to make this work, even if I was able to properly run my feature test.

Reply:
Did you define a step?

Reply:
Ok. I probably find it here
```diff
--- a/features/step_definitions/additional_cli_steps.rb
+++ b/features/step_definitions/additional_cli_steps.rb
@@ -6,6 +6,10 @@ begin
   require "action_cable"
 rescue LoadError # rubocop:disable Lint/SuppressedException
 end
+begin
+  require "action_mailbox"
+rescue LoadError # rubocop:disable Lint/SuppressedException
+end